src/dashboard/dashboard.js

```diff
@@ -28,6 +30,14 @@ import 'material-design-icons-iconfont/dist/material-design-icons.css';
   Vue.filter('friendlyShortDuration', formatSecsShort);
   Vue.filter('fixed', (v, precision) => parseFloat(v).toFixed(precision));
 
+  const messages = {
+    tip: {
+      ignore:
+        "This creator won't receive donations and won't show up among your favorite creators",
+    },
+  };
+  Vue.prototype.$t = path => _.get(messages, path);
```

Comment:
What's going on here?

Edit: nvm, noticed `$t` is how things are done by e.g. `vue-i18n`: http://kazupon.github.io/vue-i18n/guide/started.html